util/trilateration.js

```diff
@@ -0,0 +1,126 @@
+// Created by Derrick Cohodas (dav-)
+// Based on the Python example by StackExchange user wwnick from http://gis.stackexchange.com/a/415/41129
+
+
+// Requires the Mathjs library - http://mathjs.org/
+const math = require('mathjs');
+
+/**
+ * Represents a coordinate with a distance
+ * @param {Number} lat  Latitude
+ * @param {Number} lon  Longitude
+ * @param {Number} dist Distance from [lat, lon] to some point in kilometers
```

Comment:
Thanks to proximy, we know where all beacons are (with their latitudes and longitudes).
The phone gives us a list of distance between detected beacons and the phone himself

So this method returns the approximative position of the user